spec/test-utils/webrtcReports.ts

```diff
@@ -0,0 +1,1089 @@
+export const prevChromeReport = [
```

Comment:
Since these declarations have no explicit types, a comment explaining what they are would be appreciated (I assume they're example reports from the WebRTC stats API?)